docs/sources/next/misc/usage-collection.md

```diff
@@ -17,49 +19,11 @@ The usage report does not contain any information about what you are testing. Th
 - VU iterations configured (number)
 - The running program's operating system target (`darwin`, `freebsd`, `linux`...)
 - The running program's architecture target (386, amd64, arm, s390x...)
+- The list of JavaScript imported modules (`k6/http`, `k6/experimental/webcrypto`, ...)
+- The list of used outputs (`json`, `influxdb`, ...)
 
-This info is sent to an HTTP server that collects statistics on k6 usage.
-
-For those interested, here is the actual Go [code](https://github.com/grafana/k6/blob/master/cmd/run.go) that generates and sends the usage report:
-
-{{< code >}}
-
-```go
-  // If the user hasn't opted out: report usage.
-
-  if !conf.NoUsageReport.Bool {
-    go func() {
-      u := "http://k6reports.k6.io/"
-      mime := "application/json"
-      var endTSeconds float64
-
-      if endT := engine.Executor.GetEndTime(); endT.Valid {
-        endTSeconds = time.Duration(endT.Duration).Seconds()
-      }
-
-      var stagesEndTSeconds float64
-      if stagesEndT := lib.SumStages(engine.Executor.GetStages()); stagesEndT.Valid {
-        stagesEndTSeconds = time.Duration(stagesEndT.Duration).Seconds()
-      }
-
-      body, err := json.Marshal(map[string]interface{}{
-        "k6_version":  Version,
-        "vus_max":     engine.Executor.GetVUsMax(),
-        "iterations":  engine.Executor.GetEndIterations(),
-        "duration":    endTSeconds,
-        "st_duration": stagesEndTSeconds,
-        "goos":        runtime.GOOS,
-        "goarch":      runtime.GOARCH,
-      })
+> Only k6 built-in JavaScript modules and outputs are considered. Private modules and custom extensions are excluded.
 
-      if err != nil {
-        panic(err) // This should never happen!!
-      }
-      if _, err := http.Post(u, mime, bytes.NewBuffer(body)); err != nil {
-        log.WithError(err).Debug("Couldn't send usage blip")
-      }
-    }()
-  }
-```
+This report is sent to an HTTP server that collects statistics on k6 usage.
```

Comment:
I assume the statistics collection is made over HTTPs (🤞🏻) If so, I would use this opportunity to explicit that the data is transported over TLS (to ensure the confidentiality and security of our users) 🙇🏻

Reply:
It makes sense to me :+1: 

```suggestion
This report is sent to an HTTPS server that collects statistics on k6 usage.
```
@oleiade Is this change enough or were you thinking of something different?

Reply:
That works indeed, I don't necessarily think we need to go beyond that 👍🏻 Thanks for doing it 🙇🏻